code/acting/src/acting/DummyTrajectoryPublisher.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+"""
+This node publishes a dummy trajectory between predefined points.
+"""
+
+import ros_compatibility as roscomp
+from nav_msgs.msg import Path
+from geometry_msgs.msg import PoseStamped
+from ros_compatibility.node import CompatibleNode
+from trajectory_interpolation import interpolate_route
+import rospy
+
+
+class DummyTrajectoryPub(CompatibleNode):
+    """
+    Creates a node that publishes an interpolated trajectory between
+    predefined points as a nav_msgs/Path message.
+    """
+
+    def __init__(self):
+        """
+        Constructor
+        :return:
+        """
+        super(DummyTrajectoryPub, self).__init__('dummy_trajectory_pub')
+        self.loginfo('DummyTrajectoryPub node started')
+        # basic info
+        self.role_name = self.get_param('role_name', 'ego_vehicle')
+        self.control_loop_rate = self.get_param('control_loop_rate', 0.05)
+
+        self.current_trajectory = []
+        self.path_msg = Path()
+        self.path_msg.header.stamp = rospy.Time.now()
+        self.path_msg.header.frame_id = "Frame ID Path"
```

Comment:
tbh I'm not sure what the frame_id field is for either :/
["Frame this data is associated with"](http://docs.ros.org/en/noetic/api/std_msgs/html/msg/Header.html) doesn't help me either.
I therefore just set frame_id to a static string that could be traced back to the origin (Path).

Apart from that, the header should be implemented correctly (time stamps).

Reply:
I think this is related to the frame aka "Koordinatensystem" this might be helpful when using mutiple KOS for example for the map and the car driving in the map. ROS packages lik TF or TF2 can transform from one KOS(frame) to another without the programmer needing to spend much thought about it.